hack/cluster-deploy.sh

```diff
@@ -74,6 +74,14 @@ until _kubectl get crd kubevirts.kubevirt.io; do
     sleep 1
 done
 
+# Ensure the KubeVirt API is available
+count=0
+until _kubectl api-resources --api-group=kubevirt.io | grep kubevirts; do
```

Comment:
Just curious: I'm wondering what the difference is compared to the CRD check above this? 
Is it possible that `get crd` succeeds but `api-resources` fails? :thinking:

Reply:
Yes! That surprised me too but on my machine there's a ~10 second delay on average between `get crd` succeeding and `api-resources` listing anything.
The way it was explained to me is that after the CRD is created, virt-operator then has to wake up, notice it, and create the right stuff to make it available.

Reply:
As far as I know 10 seconds is within the normal boundary until new CRs are properly registered and available. Did you consider watching the CRD status? It should tell you explicitly when it is fully registered.

Reply:
That is what I was trying to do, but I couldn't figure out the syntax to get that info from `kubectl get crd`. My last attempt was this horrible output template: `{{.status.conditions[1].type}}`, which isn't even valid... Please let me know what the right command should look like and I'll fix my commit! Thank you.

Reply:
why not loop on trying to create the CR? It should be eventually consistent and pass once the CRD is registered. That way we're waiting on the thing we actually need to complete.

Reply:
@davidvossel I don't really like the idea of generating errors that really aren't... The loop just above mine serves a similar purpose, and at first my fix was to modify its condition, but in the end it made more sense to add a separate one.

Reply:
> Please let me know what the right command should look like

I needed something similar just a few days ago, just change it to your exact usecase
```
$ kubectl get vmi testvm -o=jsonpath='{.status.conditions[?(@.type=="Paused")].message}'
VMI was paused by user
```

Reply:
Thank you @slintes, it worked!
```
$ kubectl get crd kubevirts.kubevirt.io -o jsonpath='{.status.conditions[?(@.type=="Established")].status}'
True
```
I will test that in the script and consider opening a new PR, since this one got merged.